src/app/api/v1/relay/getRelayStatus.ts

```diff
@@ -0,0 +1,26 @@
+import { getPublicClient } from "@/lib/viem";
+import { cache } from "react";
+import { formatEther } from "viem";
+import { privateKeyToAccount } from "viem/accounts";
+
+const SPONSOR_PRIVATE_KEY = process.env.SPONSOR_PRIVATE_KEY;
+const GAS_COST = 0.001108297;
```

Comment:
is this for voting? maybe we can have a mapping of gas costs for different actions, since we are going to support a few (vote, delegate, maybe more in the future)